src/wp-includes/update.php

```diff
@@ -956,6 +956,54 @@ function wp_clean_update_cache() {
 	delete_site_transient( 'update_core' );
 }
 
+/**
+ * Deletes all contents of the temp-backup directory.
+ *
+ * @since 6.0.0
+ *
+ * @global WP_Filesystem_Base $wp_filesystem WordPress filesystem subclass.
+ */
+function wp_delete_all_temp_backups() {
+	/*
+	 * Check if there's a lock, or if currently performing an Ajax request,
+	 * in which case there's a chance we're doing an update.
```

Comment:
Avoid using "we" in code comments as it's not clear who "we" is referring to.

Reply:
```suggestion
	 * in which case there's a chance an update is running.
```